app/controllers/users.controller.js

```diff
@@ -21,3 +21,22 @@ exports.signUp = async (req, res, next) => {
     return next(errors.databaseError(DATABASE_ERROR));
   }
 };
+
+exports.signIn = async (req, res, next) => {
+  const { body } = req;
+  const { email, password } = userSignInMapper(body);
+  try {
+    const user = await userFindByEmail(email);
+    if (!user) throw errors.unauthorized(EMAIL_OR_PASSWORD_DO_NOT_MATCH);
+    const comparePassword = compareHash(user.password, password);
+    if (!comparePassword) throw errors.unauthorized(EMAIL_OR_PASSWORD_DO_NOT_MATCH);
+    const token = jwtEncode({
```

Comment:
I think you could move this to the helper with a method like encodeJwtUser()

Reply:
jwtEncode is currently in the helper folder ([app/helpers/auth.js](https://github.com/wolox-training/el-express-js/pull/7/files#diff-3cd07146227b248bec032c8f6c0b5a5df00d22b99c7620882f97c34cfbc32030) ). I have changed the payload for the serialized user.